pdbtools/pdb_selb.py

```diff
@@ -0,0 +1,223 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright 2024 Ahmed Selim Üzüm
```

Comment:
I am not sure if you can directly claim the whole copy right for this script has many of the sections are copied or based on the original `pdb-tools` project. For clarity and fairness, I suggest stating that the copyright if both from the `pdb-tools` project and yours.

opinon @JoaoRodrigues ?

Reply:
Of course! Since most scripts usually have just one author listed, I hadn't thought the copyright part could have multiple people. Also, should I update the `__author__` and `__email__`  parts as well, or only the copyright part?

Reply:
Would be nice to wrap up this one as this is a useful addition